src/iOS/POI/POIAttributesViewController.swift

```diff
@@ -153,8 +153,8 @@ class POIAttributesViewController: UITableViewController {
 					cell.title.text = NSLocalizedString("Length", comment: "")
 					cell.value.text = len >= 10
 						? String.localizedStringWithFormat(
-							NSLocalizedString("%.0f meters, %ld nodes", comment: "way length if > 10m"),
-							len,
+							NSLocalizedString("%ld meters, %ld nodes", comment: "way length if > 10m"),
+							Int(len),
```

Comment:
Without doing this it seems to pick the pluralization of "98.264305" instead of "98".

This would format "1.2 apples" as "1 apples", although this can't happen for English since `len >= 10`, but with Russian plurals this actually happens.